docs/src/components/landing-page/SearchQuasarForm.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <q-form
+    v-if="showSearchInputField || isOpenByDefault"
+    autocapitalize="off"
+    autocomplete="off"
+    spellcheck="false"
+    class="search-form position-relative"
+  >
+    <transition
+      appear
+      enter-active-class="animated fadeIn"
+      leave-active-class="animated fadeOut"
+    >
+      <q-input
+        ref="searchInputRef"
+        v-model="searchTerms"
+        :dark="!dark"
+        :bg-color="dark? 'black-12':'white-28'"
+        :input-class="`q-ml-xs text-size-16 letter-spacing-40 ${dark? 'text-black':'text-white'}`"
+        :autofocus="$q.screen.lt.lg"
+        standout
+        dense
+        debounce="300"
+        hide-bottom-space
+        placeholder="Search Quasar v2..."
+        @keydown="onSearchKeydown"
+        @focus="onSearchFocus"
+      >
+        <template #append>
+          <div
```

Comment:
This must have the look & feel of the existing one on current docs
It must look as a "floating" raised grey button, mimiking the look & feel of the key of an old keyboard